.github/workflows/regular-release.yaml

```diff
@@ -54,13 +54,8 @@ jobs:
             brew upgrade cbmc
             ```
 
-            ## Ubuntu
```

Comment:
This change overshot slightly - we shouldn't delete the headers or the instruction text, just the `dpkg` instructions for Ubuntu 18.